sdk/core/azure-core/azure/core/paging.py

```diff
@@ -33,7 +33,6 @@
 from typing import Dict, Any, List, Callable, Optional, TYPE_CHECKING  # pylint: disable=unused-import
 
 if TYPE_CHECKING:
-    from .pipeline import HttpResponse  # pylint: disable=unused-import
     from msrest.serialization import Deserializer, Model  # pylint: disable=unused-import
```

Comment:
It looks like a behavior change rather than a formatting change. Could you double check?

Reply:
good catch. i'll add this back. it's required for mypy anyway